docs/content/en/docs/clustermgmt/nodes/vmotion.md

```diff
@@ -0,0 +1,81 @@
+
+
+* **VMware Storage vMotion is not supported for EKS Anywhere nodes**
+
+  datastore `(spec.datastore)` - Defined in the `VSphereMachineConfig` section of the EKS Anywhere cluster spec file is immutable.  This value refers to the vSphere datastore that holds EKS Anywhere node vm backing store. Modifying the datastore during vMotion (storage vMotion) would require a change to this value, which is not supported.
+
+
+* **Node network configuration stability**
+
+  The IP address, subnet mask, and default gateway of each EKS Anywhere node must remain unchanged during the vMotion process. Any modifications to the IP address configuration can cause communication failures between the EKS Anywhere nodes, pods, and the control plane, leading to disruptions in EKS Anywhere  cluster operations.
+
+
+* **EKS Anywhere configuration stabiltiy**
+
+  The EKS Anywhere environment itself should remain unchanged during vMotion.  Do not perform or trigger any EKS Anywhere changes or life cycle events while performing vmotion.
+
+
+### Best practices for vMotion with EKS Anywhere clusters
+
+* **Follow VMware vMotion best practices**
+
+  * General best practices: Review VMware's general guidelines for optimal vMotion performance, such as ensuring sufficient CPU, memory, and network resources, and minimizing load on the ESXi hosts during the migration. Refer to the [VMware vMotion documentation](https://docs.vmware.com/) for details.
+
+  * VMware vMotion Networking Best Practices: Whenever possible, follow the [Networking Best Practices for VMware vMotion](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vcenterhost.doc/GUID-7DAD15D4-7F41-4913-9F16-567289E22977.html) to optimize performance and reduce the risk of issues during the migration process.
+
+  * Use High-Speed Networks: A 10GbE or higher speed network is recommended to ensure smooth vMotion operations for EKS Anywhere nodes, particularly those with large memory footprints.
+
+
+* **Shared Storage**
+
+  Shared storage is a requirement for vmotion of EKS Anywhere clusters.  Storage such as vSAN, Fiber Channel SAN, or NFS should be shared between the supporting vSphere ESXi hosts for maintaining access to the VM's backing data without relying on storage vMotion, which is not supported in EKS Anywhere environments.
+
+
+* **Monitoring before and after migration**
+
+  To verify cluster health and node stability,  monitor the EKS Anywhere nodes and pods before and after the vMotion migration:
+
+  * Before migration, run the following commands to check the current health and status of the EKS Anywhere nodes and pods.
+  * After vMotion activity is completed, run the commands again to verify that the nodes and pods are still operational and healthy.
+
+  kubectl get nodes
```

Comment:
Sorry, something weird happened here. I tried to do two spaces in front of the three backticks before and after the two kubectl lines. Somehow they got deleted. Can you add them back in around those two lines and indent them two spaces?

Reply:
restored backticks